packages/atlas/src/views/playground/Playgrounds/PlaygroundReactionsComments.tsx

```diff
@@ -0,0 +1,246 @@
+import React, { useState } from 'react'
+
+import { TabItem, Tabs } from '@/components/Tabs'
+import { Button } from '@/components/_buttons/Button'
+import { Select, SelectItem } from '@/components/_inputs/Select'
+import { TextArea } from '@/components/_inputs/TextArea'
+import { TextField } from '@/components/_inputs/TextField'
+import { VideoReaction } from '@/joystream-lib'
+import { useJoystream } from '@/providers/joystream'
+import { useTransaction } from '@/providers/transactionManager'
+import { useUser } from '@/providers/user'
+import { ConsoleLogger } from '@/utils/logs'
+
+const TABS: TabItem[] = [
+  { name: 'React to video' },
+  { name: 'React to comment' },
+  { name: 'Create comment' },
+  { name: 'Edit comment' },
+  { name: 'Delete comment' },
+]
+
+export const PlaygroundReactionsComments: React.FC = () => {
+  const [selectedTabIdx, setSelectedTabIdx] = useState(0)
+  const [videoId, setVideoId] = useState('')
+
+  const getTabContents = () => {
+    const props: CommonProps = {
+      videoId,
+    }
+    switch (selectedTabIdx) {
+      case 0:
+        return <ReactToVideo {...props} />
+      case 1:
+        return <ReactToComment {...props} />
+      case 2:
+        return <CreateComment {...props} />
+      case 3:
+        return <EditComment {...props} />
+      case 4:
+        return <DeleteComment {...props} />
+    }
+  }
+
+  return (
+    <div>
+      <TextField value={videoId} onChange={(e) => setVideoId(e.target.value)} label="Video ID" />
+      <Tabs tabs={TABS} onSelectTab={setSelectedTabIdx} />
+      {getTabContents()}
+    </div>
+  )
+}
+
+type CommonProps = {
+  videoId: string
+}
+
+const ReactToVideo: React.FC<CommonProps> = ({ videoId }) => {
+  const { joystream, proxyCallback } = useJoystream()
+  const handleTransaction = useTransaction()
+  const { activeMemberId } = useUser()
+  const [videoReaction, setVideoReaction] = useState<VideoReaction>('like')
+
+  const handleLike = () => {
+    if (!joystream || !activeMemberId) {
+      ConsoleLogger.error('no joystream or active member')
+      return
+    }
+
+    handleTransaction({
+      txFactory: async (updateStatus) =>
+        (await joystream.extrinsics).reactToVideo(activeMemberId, videoId, videoReaction, proxyCallback(updateStatus)),
+      minimized: {
+        signErrorMessage: 'Failed to react to video',
+      },
+    })
+  }
+
+  const reactionItems: SelectItem<VideoReaction>[] = [
+    { value: 'like', name: 'Like' },
+    { value: 'dislike', name: 'Dislike' },
+  ]
+
+  return (
+    <div>
+      <Select
+        value={videoReaction}
+        onChange={(value) => value && setVideoReaction(value)}
+        items={reactionItems}
+        label="Reaction type"
+      />
+      <Button onClick={handleLike}>React</Button>
+    </div>
+  )
+}
+const ReactToComment: React.FC<CommonProps> = () => {
```

Comment:
A similar question to the above. Is it possible to cancel the reaction?

Reply:
Should be the same as above